src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngine.java

```diff
@@ -47,6 +48,44 @@ private VariantAnnotatorEngine(final AnnotationManager annots,
         }
     }
 
+
+    /**
+     * * Makes the engine given a list of annotations generated by the the command line parser.
+     * @param annotationList List of annotation objects with user argument parameters already set which should be applied
+     *                       by the engine when annotating VariantContexts
+     * @param dbSNPInput input for variants from a known set from DbSNP or null if not provided.
+     *                   The annotation engine will mark variants overlapping anything in this set using {@link htsjdk.variant.vcf.VCFConstants#DBSNP_KEY}.
+     * @param featureInputs list of inputs with known variants.
+     *                   The annotation engine will mark variants overlapping anything in those sets using the name given by {@link FeatureInput#getName()}.
+     *                   Note: the DBSNP FeatureInput should be passed in separately, and not as part of this List - an GATKException will be thrown otherwise.
+     *                   Note: there are no non-DBSNP comparison FeatureInputs an empty List should be passed in here, rather than null.
+     * @param useRaw Boolean value indicating whether the engine should output its reducible annotations in their raw form
+     */
+    public VariantAnnotatorEngine(final List<Annotation> annotationList,
+                                  final FeatureInput<VariantContext> dbSNPInput,
+                                  final List<FeatureInput<VariantContext>> featureInputs,
+                                  final boolean useRaw){
```

Comment:
Yes, Essentially the next branch needed a use-raw argument but I chose not to hook it up. It would be a hassle to change all the tests that rely on this method.